CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### ⚡ Performance
 - Improve message search performance [#680](https://github.com/GetStream/stream-chat-swiftui/pull/680)
+### 🔄 Changed
+- Make `CreatePollView` public [#685](https://github.com/GetStream/stream-chat-swiftui/pull/685)
```

Comment:
This should actually be an addition 🤔 We are exposing a new view

Reply:
yep